### ISSUE TITLE
Allow some functions to be filtered

### ISSUE THREAD

**Issue by zedzedzed** [zedzedzed/table-of-contents-plus#79]
**Reported by michael on 24 Feb 2013 08:57 UTC**
url_anchor_target to allow hyphens rather than underscores

**Comment by zedzedzed** [comment]
**Modified by michael on 18 Mar 2013 07:54 UTC**

**Comment by zedzedzed** [comment]
**Comment by michael on 19 Mar 2013 12:09 UTC**
In [changeset:"219"]:
```CommitTicketReference repository="" revision="219"
closes #84
also adds new options to lowercase and hyphenate anchors for those that aren't able to apply PHP changes
```

**Comment by zedzedzed** [comment]
**Modified by michael on 21 Mar 2013 13:00 UTC**
